lib/tasks/import/api.js

```diff
@@ -150,7 +150,7 @@ async function downloadContentExport(version, url, auth, outputFile) {
 
 async function downloadMembersExport(version, url, auth, outputFile) {
     const authOpts = await getAuthOpts(version, url, auth);
-    let endpoint = '/members/upload/';
+    let endpoint = '/members/upload?limit=all';
```

Comment:
```suggestion
    let endpoint = '/members/upload/?limit=all';
```

I think it's best to leave the slash otherwise it'll lead to a redirect because we enforce slashes